aptos-move/move-examples/fungible_asset/coin_wrapper/sources/coin_wrapper.move

```diff
@@ -0,0 +1,220 @@
+/// This module can be included in a project to enable internal wrapping and unwrapping of coins into fungible assets.
+/// This allows the project to only have to store and process fungible assets in core data structures, while still be
+/// able to support both native fungible assets and coins. Note that the wrapper fungible assets are INTERNAL ONLY and
+/// are not meant to be released to user's accounts outside of the project. Othwerwise, this would create multiple
+/// conflicting fungible asset versions of a specific coin in the ecosystem.
```

Comment:
for EXTERNAL USE do we still need to count on the coin module?

Reply:
Yes that'd be the main flow for the original coin creator to use to create a matching fungible asset (in terms of mint/burn permissions and exact 1:1 mapping). This way there would be at most one fungible asset representation of the coin. This module is only for internal use for a project and if each creator deploys this outside, there would be multiple versions of fungible assets for a coin as explained here

Reply:
@movekevin i am curious under what scenario would users prefer the internal module from coin <> fungible asset conversion, over the coin module?

Reply:
This is required to supports coins that havent migrated